### DISCUSSION TITLE
Select single value from query response in Handlebars binding

### DISCUSSION THREAD

**Discussion by chrisshaw** [Budibase/budibase#2533] [Help] [answered]
**How can I bind a single value from a query to a Stat card?**

Surely this _can't_ be as hard as I'm making it. I'm sure I'm doing something rudimentary.

I have a PostgreSQL custom query that returns the following:

```
{
  "messages": "1685",
  "conversations": "260"
}
```

![image](https://user-images.githubusercontent.com/2966918/131934255-05947a7e-b88a-4c75-b4e6-9d50907fd59d.png)

I have set a Data Provider to use this Query. Then, inside that Data Provider, I have a Stat card element I would like to fill with the "messages" property from this query.

![image](https://user-images.githubusercontent.com/2966918/131934535-f7943b11-3149-44aa-8dc9-337c363a8231.png)

I cannot for the life of me figure it out. Meanwhile the Table component will drop in this single row just fine!

**Comment by shogunpurple** [comment, marked as answer]
Hi @chrisshaw, you should be able to do this with the following bindings:

```
{{ New Data Provider.Rows.0.messages }}
{{ New Data Provider.Rows.0.conversations }}
```

**Reply by chrisshaw** [reply]
Thanks! I think what threw me is the Preview from the query shows it as a top level object. Does the Data Provider component wrap it or does it actually come back in an array from the query?
